lib/llm-events/openai/event.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2023 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+const { makeId } = require('../../util/hashes')
+const { DESTINATIONS } = require('../../../lib/config/attribute-filter')
+
+module.exports = class LlmEvent {
+  constructor({ agent, segment, request, response, responseAttrs = false }) {
+    this.agent = agent
+    this.id = makeId(36)
+    this.appName = agent.config.applications()[0]
+    this.request_id = response?.headers?.['x-request-id']
+    this.trace_id = segment?.transaction?.traceId
+    this.span_id = segment?.id
+    this.transaction_id = segment?.transaction?.id
+    this.metadata = agent?.llm?.metadata
+    this['response.model'] = response.model
+    this.vendor = 'openAI'
+    this.ingest_source = 'Node'
+
+    // used in embedding, chat completion summary
+    // inclues response attributes but other attrs
+    // like api key and duration
```

Comment:
I'm having a hard time following this comment. As I understand it, everything in the `if` block is used by the "embedding" and "chat completion summary" APIs. But what are lines 2 and 3 saying?
```suggestion
    // used in embedding, chat completion summary
    // includes response attributes but other attrs
    // like api key and duration
```

Reply:
ok yea, so maybe I need to tweak.  so the property is `responseAttrs` it's a flag to set more data on the event.  It's actually more than data from response. They are used in embedding and chat completion summary.  However the request model, api key and duration are also things that get set on those events.  I'm def open to making this comment clearer